elasticsearch/sgconfig/action_groups.yml

```diff
@@ -45,6 +45,8 @@ READ:
 
 DELETE:
   - "indices:data/write/delete*"
+  - "indices:data/write/bulk*"
+  - "indices:data/read/scroll*"
```

Comment:
Should these permissions just be given to the specific `sg_curator` role or to anyone adopting this action? I see that `DELETE` is a part `INDEX_KIBANA`.

Reply:
We should give the permission to this group since delete by query can be run from kibana as well like this https://github.com/openshift/elasticsearch-operator/blob/master/docs/troubleshooting.md#delete-logs-for-specific-namespace

Reply:
I see. Thanks!